src/test/java/uk/gov/hmcts/reform/civil/service/robotics/mapper/EventHistoryMapperTest.java

```diff
@@ -854,6 +854,186 @@ void shouldPrepareExpectedEvent_whenRespondent1ExtendsTimeRespondent2TimeExtende
             );
              **/
         }
+
+        @Test
+        void shouldPrepareExpectedEvent_whenRespondent1AcknowledgesRespondent2AlreadyAcknowledged() {
```

Comment:
If you move that Acknowledgement date determination logic outside, these tests can specifically target those if condition rather thanadding the complication of testing with the EventHistoryMApper. Please consider this now or probably leave a TODO to do this after release 1.5